### DISCUSSION TITLE
What does the snowflake mean

### DISCUSSION THREAD

**Discussion by denzel1994** [teslamate-org/teslamate#2255] [Q&A]
Hey there,

can someone help me and explain, what the snowflake in the „Drives“ overview mean?
![22ED5969-DF0F-4707-B8A7-0BCF01D5D133](https://user-images.githubusercontent.com/36551243/147832823-8755791e-53da-4875-b80b-fa0a7218b9a0.jpeg)

12 degrees Celsius isn’t quite freezing.

And why are „km driven“ in „Drive Stats“ not showing?
![362582EB-BB2C-489B-B7C9-74D5B3847C7B](https://user-images.githubusercontent.com/36551243/147832854-a19445c1-1d30-4760-8dd6-b0b77437fbf2.jpeg)

Thank you. :)

**Comment by djpalmis** [comment]
I thought I was the only one who didn't understand what that symbol means..

**Comment by martinh2011** [comment]
If I understand the SQL query behind that dashboard correctly, it works like this: The Tesla-API provides a „state of charge“ (% SoC) level and a „usable SoC“ level which might be lower, e.g. when the battery is cold. The snowflake is shown when these two values differ, measured either at the start or at the end of the drive.
